package.json

```diff
@@ -164,6 +166,7 @@
 					"requireEndingPeriod": true
 				}
 			],
+			"prefer-no-devDependencies": "error",
```

Comment:
There is new rule added which we will benefit from in Gutenberg. See https://github.com/tclindner/npm-package-json-lint/pull/133. Props to @simison for working on it 💯